resources/assets/js/app.vue

```diff
@@ -280,54 +280,52 @@
      *
      * @source https://github.com/vuejs/vue/blob/dev/src/filters/array-filters.js
      */
-    Vue.filter('caseInsensitiveOrderBy', (arr, sortKey, reverse, subSortKey) => {
+    Vue.filter('caseInsensitiveOrderBy', (arr, sortKey, reverse) => {
         if (!sortKey) {
             return arr;
         }
 
         let order = (reverse && reverse < 0) ? -1 : 1;
 
+        function compareRecordsByKey(a, b, key) {
+            let aKey = Vue.util.isObject(a) ? Vue.parsers.path.getPath(a, key) : 0;
+            let bKey = Vue.util.isObject(b) ? Vue.parsers.path.getPath(b, key) : 0;
+
+            if (_.isNumber(aKey) && _.isNumber(bKey)) {
+                return aKey === bKey ? 0 : aKey > bKey;
+            }
+
+            aKey = aKey === undefined ? aKey : aKey.toLowerCase();
```

Comment:
The problem is that we're not setting the key check to the variable itself as we are when only using 1 key because we need to preserve the object. I've corrected this. Let me know if the issue is resolved for you (I wasn't able to replicate the issue).